roles/common/tasks/monitoring.yml

```diff
@@ -5,6 +5,9 @@
 - name: install sensu
   apt: pkg=sensu
 
+- name: install libfile-which-perl
+  apt: pkg=libfile-which-perl
+
```

Comment:
I hate to be nit-picky, but could you turn the task just before this into a with_items task and list sensu and libfile-which-perl as items to be installed? It's a small efficiency thing but something I'd do as a later clean up anyway.